src/blocks/block-post-grid/index.php

```diff
@@ -28,15 +28,15 @@ function atomic_blocks_render_block_core_latest_posts( $attributes ) {
 		$post_thumb_id = get_post_thumbnail_id( $post_id );
 
 		if ( $post_thumb_id && isset( $attributes['displayPostImage'] ) && $attributes['displayPostImage'] ) {
-			$post_thumb_class = 'has-thumb';
+			$post_classes = get_post_class( 'has-thumb', $post->ID );
 		} else {
-			$post_thumb_class = 'no-thumb';
+			$post_classes = get_post_class( 'no-thumb', $post->ID );
 		}
 
 		// Start the markup for the post
 		$list_items_markup .= sprintf(
-			'<article class="%1$s">',
-			esc_attr( $post_thumb_class )
+			'<article id="post-'.$post_id.'" class="%1$s">',
```

Comment:
We should escape this value. It's unlikely to be anything other than an integer, but since so much of WP is filterable, we want to be sure we're being as safe as possible.

Reply:
On line 25, $post_id is set from $post->ID. How can it be anything but an integer?

Reply:
I mentioned it's unlikely, but we escape all dynamic output.